### PR TITLE
DRDT-86: Port Taboola to DRDT project

### PR DESCRIPTION
**Pull request by fariasf**
Instead of hardcoding the Publisher ID and widget IDs, this implementation
uses Widgets to render Taboola into the page. This way it can be used in many
sites (by changing the `taboola_publisher_id` option) and with different
configurations (each widget takes the `mode`, `placement`, and `target_type` params).

*Todo:*
- [x] Rename the plugin to avoid conflicts with the official Taboola plugin
- [x] Add settings screen so that we don't need `wp cli` to change the Publisher ID
- [x] Store default config in WP CFM
- [x] Add option to remove Taboola via querystring (react to `noads` and other 3rd-party blocking mechanisms)
- [x] Remove jQuery dependency

### CODE REVIEW

**Comment by fariasf** [comment]
Created new staging environment for pr-71. Here are some useful links:

[WordPress Admin](https://pr-71-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-71-drdt.pantheonsite.io/)
[Article](https://pr-71-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-71-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-71-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-71-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2357/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
⚠ Frontend performance tests failed!

Full Lighthouse performance results:

| Page/Branch | pr-71 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [92](https://circleci.com/gh/fariasf/drdt-test/2356/artifacts/0/artifacts/lighthouse_results/pr-71/homepage/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2356/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [98](https://circleci.com/gh/fariasf/drdt-test/2356/artifacts/0/artifacts/lighthouse_results/pr-71/article/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2356/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [96](https://circleci.com/gh/fariasf/drdt-test/2356/artifacts/0/artifacts/lighthouse_results/pr-71/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2356/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [89](https://circleci.com/gh/fariasf/drdt-test/2356/artifacts/0/artifacts/lighthouse_results/pr-71/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2356/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✘ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2356/artifacts/0/artifacts/lighthouse_results/pr-71/hub/lighthouse.report.html) | [95](https://circleci.com/gh/fariasf/drdt-test/2356/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-71. Here are some useful links:

[WordPress Admin](https://pr-71-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-71-drdt.pantheonsite.io/)
[Article](https://pr-71-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-71-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-71-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-71-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2363/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Congrats! Frontend performance tests passed!

Full Lighthouse performance results:

| Page/Branch | pr-71 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [96](https://circleci.com/gh/fariasf/drdt-test/2362/artifacts/0/artifacts/lighthouse_results/pr-71/homepage/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2362/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [98](https://circleci.com/gh/fariasf/drdt-test/2362/artifacts/0/artifacts/lighthouse_results/pr-71/article/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2362/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2362/artifacts/0/artifacts/lighthouse_results/pr-71/listicle/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2362/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [96](https://circleci.com/gh/fariasf/drdt-test/2362/artifacts/0/artifacts/lighthouse_results/pr-71/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2362/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2362/artifacts/0/artifacts/lighthouse_results/pr-71/hub/lighthouse.report.html) | [94](https://circleci.com/gh/fariasf/drdt-test/2362/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-71. Here are some useful links:

[WordPress Admin](https://pr-71-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-71-drdt.pantheonsite.io/)
[Article](https://pr-71-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-71-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-71-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-71-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2376/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Congrats! Frontend performance tests passed!

Full Lighthouse performance results:

| Page/Branch | pr-71 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [96](https://circleci.com/gh/fariasf/drdt-test/2377/artifacts/0/artifacts/lighthouse_results/pr-71/homepage/lighthouse.report.html) | [94](https://circleci.com/gh/fariasf/drdt-test/2377/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [97](https://circleci.com/gh/fariasf/drdt-test/2377/artifacts/0/artifacts/lighthouse_results/pr-71/article/lighthouse.report.html) | [94](https://circleci.com/gh/fariasf/drdt-test/2377/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2377/artifacts/0/artifacts/lighthouse_results/pr-71/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2377/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2377/artifacts/0/artifacts/lighthouse_results/pr-71/page/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2377/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2377/artifacts/0/artifacts/lighthouse_results/pr-71/hub/lighthouse.report.html) | [92](https://circleci.com/gh/fariasf/drdt-test/2377/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-71. Here are some useful links:

[WordPress Admin](https://pr-71-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-71-drdt.pantheonsite.io/)
[Article](https://pr-71-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-71-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-71-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-71-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2383/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
⚠ Frontend performance tests failed!

Full Lighthouse performance results:

| Page/Branch | pr-71 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [80](https://circleci.com/gh/fariasf/drdt-test/2384/artifacts/0/artifacts/lighthouse_results/pr-71/homepage/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2384/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✘ |
| Article | [89](https://circleci.com/gh/fariasf/drdt-test/2384/artifacts/0/artifacts/lighthouse_results/pr-71/article/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2384/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✘ |
| Listicle | [92](https://circleci.com/gh/fariasf/drdt-test/2384/artifacts/0/artifacts/lighthouse_results/pr-71/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2384/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✘ |
| Page | [93](https://circleci.com/gh/fariasf/drdt-test/2384/artifacts/0/artifacts/lighthouse_results/pr-71/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2384/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [91](https://circleci.com/gh/fariasf/drdt-test/2384/artifacts/0/artifacts/lighthouse_results/pr-71/hub/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2384/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✘ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-71. Here are some useful links:

[WordPress Admin](https://pr-71-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-71-drdt.pantheonsite.io/)
[Article](https://pr-71-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-71-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-71-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-71-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2389/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Congrats! Frontend performance tests passed!

Full Lighthouse performance results:

| Page/Branch | pr-71 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [96](https://circleci.com/gh/fariasf/drdt-test/2390/artifacts/0/artifacts/lighthouse_results/pr-71/homepage/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2390/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [98](https://circleci.com/gh/fariasf/drdt-test/2390/artifacts/0/artifacts/lighthouse_results/pr-71/article/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2390/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [97](https://circleci.com/gh/fariasf/drdt-test/2390/artifacts/0/artifacts/lighthouse_results/pr-71/listicle/lighthouse.report.html) | [93](https://circleci.com/gh/fariasf/drdt-test/2390/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2390/artifacts/0/artifacts/lighthouse_results/pr-71/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2390/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [96](https://circleci.com/gh/fariasf/drdt-test/2390/artifacts/0/artifacts/lighthouse_results/pr-71/hub/lighthouse.report.html) | [94](https://circleci.com/gh/fariasf/drdt-test/2390/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-71. Here are some useful links:

[WordPress Admin](https://pr-71-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-71-drdt.pantheonsite.io/)
[Article](https://pr-71-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-71-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-71-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-71-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2396/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Congrats! Frontend performance tests passed!

Full Lighthouse performance results:

| Page/Branch | pr-71 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [91](https://circleci.com/gh/fariasf/drdt-test/2397/artifacts/0/artifacts/lighthouse_results/pr-71/homepage/lighthouse.report.html) | [95](https://circleci.com/gh/fariasf/drdt-test/2397/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [98](https://circleci.com/gh/fariasf/drdt-test/2397/artifacts/0/artifacts/lighthouse_results/pr-71/article/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2397/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [97](https://circleci.com/gh/fariasf/drdt-test/2397/artifacts/0/artifacts/lighthouse_results/pr-71/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2397/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2397/artifacts/0/artifacts/lighthouse_results/pr-71/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2397/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [96](https://circleci.com/gh/fariasf/drdt-test/2397/artifacts/0/artifacts/lighthouse_results/pr-71/hub/lighthouse.report.html) | [94](https://circleci.com/gh/fariasf/drdt-test/2397/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-71. Here are some useful links:

[WordPress Admin](https://pr-71-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-71-drdt.pantheonsite.io/)
[Article](https://pr-71-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-71-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-71-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-71-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2403/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Congrats! Frontend performance tests passed!

Full Lighthouse performance results:

| Page/Branch | pr-71 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [96](https://circleci.com/gh/fariasf/drdt-test/2404/artifacts/0/artifacts/lighthouse_results/pr-71/homepage/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2404/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [98](https://circleci.com/gh/fariasf/drdt-test/2404/artifacts/0/artifacts/lighthouse_results/pr-71/article/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2404/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2404/artifacts/0/artifacts/lighthouse_results/pr-71/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2404/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2404/artifacts/0/artifacts/lighthouse_results/pr-71/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2404/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [96](https://circleci.com/gh/fariasf/drdt-test/2404/artifacts/0/artifacts/lighthouse_results/pr-71/hub/lighthouse.report.html) | [94](https://circleci.com/gh/fariasf/drdt-test/2404/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-71. Here are some useful links:

[WordPress Admin](https://pr-71-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-71-drdt.pantheonsite.io/)
[Article](https://pr-71-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-71-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-71-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-71-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2411/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Congrats! Frontend performance tests passed!

Full Lighthouse performance results:

| Page/Branch | pr-71 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [97](https://circleci.com/gh/fariasf/drdt-test/2410/artifacts/0/artifacts/lighthouse_results/pr-71/homepage/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2410/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [97](https://circleci.com/gh/fariasf/drdt-test/2410/artifacts/0/artifacts/lighthouse_results/pr-71/article/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2410/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2410/artifacts/0/artifacts/lighthouse_results/pr-71/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2410/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2410/artifacts/0/artifacts/lighthouse_results/pr-71/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2410/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2410/artifacts/0/artifacts/lighthouse_results/pr-71/hub/lighthouse.report.html) | [93](https://circleci.com/gh/fariasf/drdt-test/2410/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Review by angelicagligich — APPROVED** [review]

**Comment by fariasf** [comment]
Updated staging environment for pr-71. Here are some useful links:

[WordPress Admin](https://pr-71-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-71-drdt.pantheonsite.io/)
[Article](https://pr-71-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-71-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-71-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-71-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2445/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Congrats! Frontend performance tests passed!

Full Lighthouse performance results:

| Page/Branch | pr-71 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [96](https://circleci.com/gh/fariasf/drdt-test/2446/artifacts/0/artifacts/lighthouse_results/pr-71/homepage/lighthouse.report.html) | [95](https://circleci.com/gh/fariasf/drdt-test/2446/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [98](https://circleci.com/gh/fariasf/drdt-test/2446/artifacts/0/artifacts/lighthouse_results/pr-71/article/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2446/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [97](https://circleci.com/gh/fariasf/drdt-test/2446/artifacts/0/artifacts/lighthouse_results/pr-71/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2446/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2446/artifacts/0/artifacts/lighthouse_results/pr-71/page/lighthouse.report.html) | [81](https://circleci.com/gh/fariasf/drdt-test/2446/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2446/artifacts/0/artifacts/lighthouse_results/pr-71/hub/lighthouse.report.html) | [92](https://circleci.com/gh/fariasf/drdt-test/2446/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

File: web/wp-content/plugins/tmbi-taboola/js/taboola_loader.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+(function (e, f, u) {
+	e.async = 1;
+	e.src = u;
+	f.parentNode.insertBefore(e, f);
+})(
+	document.createElement('script'),
+	document.getElementsByTagName('script')[0],
+	tmbi_taboola.script
+);
+document.addEventListener("DOMContentLoaded", function(){
```
Comment (subinviswanathan) [inline web/wp-content/plugins/tmbi-taboola/js/taboola_loader.js:13]: @fariasf could have been better if used load instead of DOMContentloaded

**Pull request merged**